ansible/roles/fastpath/tasks/main.yml

```diff
@@ -83,28 +97,67 @@
   template:
     src: templates/fastpath.conf
     dest: "/opt/{{fastpath_user}}/backend/fastpath/fastpath.conf"
-    mode: 0444
+    mode: "0700"
```

Comment:
Does this file need the executable bit set? Does it need to be writable? Generally we keep permissions of config files to read-only so that you are reminded to not edit them by hand on the host.